plotting/comparison.py

```diff
@@ -122,7 +125,26 @@
 if args.shortcomp == "":
     args.shortcomp = args.compared
 
-for discr in args.variable.split(","):
+if args.variable == "all":
```

Comment:
Can you mention this feature in the readme as well?

Reply:
Would also add one option `*` to plot the variables containing sub-string in the variable. e.g. "*DeepJet*"

Reply:
:+1: do the same for plotdataMC.py as well, and add this feature in the readme